### PR TITLE
Beaker fix

### PR DESCRIPTION
**Pull request by aopincar**

### CODE REVIEW

**Comment by aopincar** [comment]
retest openstack

**Comment by aopincar** [comment]
retest openstack

**Comment by aopincar** [comment]
retest openstack

**Comment by aopincar** [comment]
retest openstack

**Comment by aopincar** [comment]
retest openstack

File: settings/provisioner/virsh/virsh.yml [hunk 1]
```diff
@@ -23,6 +23,7 @@ provisioner:
         - xorg-x11-apps
         - xauth
         - virt-viewer
+        - wget
```
Comment (fhubik) [inline settings/provisioner/virsh/virsh.yml]: I don't think this should be part of this PR as this is "Beaker fix" not virsh, but it is definitely needed, though ;)

File: library/beaker_provisioner.py [hunk 2]
```diff
@@ -343,8 +343,9 @@ def _wait_for_provision(self, pre_provision_activity_id, timeout):
         while (time.time() - start_time) < timeout:
             time.sleep(WAIT_BETWEEN_PROVISION_CHECKS)
             last_activity = self._get_last_system_activity()
-            if last_activity['id'] != pre_provision_activity_id['id'] \
-                    and last_activity['action'] == 'clear_netboot':
+            if all((last_activity['id'] != pre_provision_activity_id['id'],
+                    last_activity['action'] == 'clear_netboot',
+                    last_activity['service'] == 'XMLRPC')):
```
Comment (fhubik) [inline library/beaker_provisioner.py]: Huh, good catch ;)

**Comment by fhubik** [comment]
Reviewed.

**Pull request merged**